mender-auth/ipc/server.hpp

```diff
@@ -54,13 +54,14 @@ class Caching {
 		client_config_ {config.server_certificate},
 		client_ {client_config_, loop},
 		server_config_ {},
-		server_ {server_config_, loop} {};
+		server_ {server_config_, loop},
+		default_identity_script_path {config.default_config.GetDefaultIdentityScript()} {};
 
 	Caching(Caching &&) = default;
 
 	error::Error Listen(
 		const string &server_url,
-		const string &identity_script_path = conf::paths::DefaultIdentityScript,
+		const string &identity_script_path = "",
```

Comment:
This shouldn't have a default if we can't provide one, or? Unless you want to use `default_identity_script_path`, but it seems unused.